src/main/java/pro/freeserver/plugin/infrastructure/resources/EconomyRepository.kt

```
@@ -122,4 +122,33 @@ class EconomyRepository(private val client: SupabaseClient) {
+            val response = client
+                .postgrest
+                .rpc(
+                    "withdraw_wallet",
+                    JsonObject(
+                        mapOf(
+                            "target_mcuuid" to JsonPrimitive(uuid.toString()),
+                            "withdraw_balance" to JsonPrimitive(balance)
+                        )
+                    )
+                ).data
+            if (response == "null") throw NullPointerException()
```

detekt (1.23.7)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)